packages/manager/apps/pci-volume-backup/src/pages/listing/Actions.component.tsx

```diff
@@ -12,38 +11,34 @@ type ActionsProps = {
 export default function Actions({ backup }: Readonly<ActionsProps>) {
   const { t } = useTranslation('listing');
 
-  const isRestoreVolumeActionAvailable = useMemo(
-    () => [VOLUME_BACKUP_STATUS.OK].includes(backup?.status),
-    [backup],
-  );
+  const restoreBackupItem = {
+    id: 1,
+    href: useHref(`./restore-volume/${backup.volumeId}`),
+    label: t(
+      'pci_projects_project_storages_volume_backup_list_datagrid_menu_action_restore',
+    ),
+  };
+  const createVolumeItem = {
+    id: 2,
+    href: useHref(`./${backup.id}/create-volume`),
+    label: t(
+      'pci_projects_project_storages_volume_backup_list_datagrid_menu_action_create_volume',
+    ),
+  };
+  const deleteVolumeItem = {
+    id: 3,
+    href: useHref(`./delete/${backup.id}`),
+    label: t(
+      'pci_projects_project_storages_volume_backup_list_datagrid_menu_action_delete',
+    ),
+  };
 
-  const items: { id: number; href: string; label: string }[] = [
-    ...(isRestoreVolumeActionAvailable
-      ? [
-          {
-            id: 1,
-            href: useHref(`./restore-volume/${backup.volumeId}`),
-            label: t(
-              'pci_projects_project_storages_volume_backup_list_datagrid_menu_action_restore',
-            ),
-          },
-        ]
-      : []),
-    {
-      id: 2,
-      href: useHref(`./${backup.id}/create-volume`),
-      label: t(
-        'pci_projects_project_storages_volume_backup_list_datagrid_menu_action_create_volume',
-      ),
-    },
-    {
-      id: 3,
-      href: useHref(`./delete/${backup.id}`),
-      label: t(
-        'pci_projects_project_storages_volume_backup_list_datagrid_menu_action_delete',
-      ),
-    },
-  ];
+  let items: { id: number; href: string; label: string }[] = [];
```

Comment:
There are other statuses for this item (restoring, deleting, creating, ok, error:

```
export type TBackupStatus =
  | 'creating'
  | 'deleting'
  | 'error'
  | 'ok'
  | 'restoring'
  | string;
```

Maybe you should provide a fallback for the other cases